Resources/Prototypes/Guidebook/ss14.yml

```diff
@@ -14,3 +14,8 @@
   id: Writing
   name: guide-entry-writing
   text: "/ServerInfo/Guidebook/Writing.xml"
+
+- type: guideEntry
```

Comment:
Considering the relevance of DA RULES. Might be a good idea to bump this all the way up to the top or just below basic controls.

Reply:
It is at the top, prototype ordering doesn't matter.